app/assets/stylesheets/active_admin/mixins/_variables.scss

```diff
@@ -2,33 +2,236 @@
+$button-default-background-color: #fff !default;
+$button-default-border-color: #d9d9d9 !default;
+
+$button-primary-color: #fff !default;
+$button-primary-background-color: $primary-color !default;
+$button-primary-border-color: darken($primary-color, 5%) !default;
+
+//== Forms
+
+$input-background-color: #fff !default;
+$input-border-color: lighten(#bdc3c7, 5%) !default;
+$input-border-focus-color: darken(#bdc3c7, 5%) !default;
+$input-border-radius: $border-radius-base !default;
+$input-color: $text-color !default;
+
+$required-field-marker-color: #aaa !default;
+
+$cursor-disabled: not-allowed !default; // for forms and buttons
+
+$form-label-width: 30%;
+$form-input-width: 70%;
+
+//== Dropdowns
+
+$caret-width-base: 4px !default;
+
+$dropdown-background-color: #fff !default;
+$dropdown-border-color: rgba(0, 0, 0, 0.15) !default;
+
+$dropdown-link-color: $text-color !default;
+$dropdown-link-hover-color: darken($text-color, 5%) !default;
+$dropdown-link-hover-background-color: #e9e9e9 !default;
+
+//== Tables
+
+$table-border-color: darken($body-background-color, 12%) !default;
+$table-cell-padding: 8px !default;
+$table-condensed-cell-padding: 5px !default;
+
+$table-stripe-color: lighten(#ecf0f1, 4%) !default;
+$table-selected-color: #fcf9c1 !default;
+
+//== Badges
+
+$badge-color: #fff !default;
+$badge-background-color: rgba(0, 0, 0, 0.35) !default;
+
+$badge-border-radius: 10px !default;
+$badge-font-weight: $font-weight-normal !default;
+$badge-line-height: 1 !default;
+
+//== Breadcrumbs
+
+$breadcrumb-color: #7f8c8d !default;
+$breadcrumb-separator-color: #7f8c8d !default;
+$breadcrumb-separator: "/" !default;
+
+//== Flash Messages
+
+$flash-message-font-weight: $font-weight-bold !default;
+$flash-message-padding: 12px $horizontal-page-margin !default;
+
+$flash-message-default-background-color: $warning-color !default;
+$flash-message-default-text-color: #fff !default;
+
+$flash-message-notice-background-color: $success-color !default;
+$flash-message-notice-text-color: #fff !default;
+
+$flash-message-error-background-color: $error-color !default;
+$flash-message-error-text-color: #fff !default;
+
+//== Modals
+
+$modal-content-padding: 15px !default;
+$modal-heading-padding: 10px 15px !default;
+
+$modal-background-color: #fff !default;
+$modal-backdrop-background-color: rgba(0, 0, 0, 0.5) !default;
+$modal-heading-border-color: #eee !default;
+
+//== Navbar
+
+$navbar-border-radius: $border-radius-base !default;
+$navbar-padding-horizontal: 10px !default;
+$navbar-padding-vertical: 10px !default;
+
+$navbar-color: #fff !default;
+$navbar-background-color: $primary-color !default;
+
+$navbar-link-color: #fff !default;
+$navbar-link-hover-color: #fff !default;
+$navbar-link-hover-background-color: lighten($primary-color, 8%) !default;
+$navbar-link-active-color: #fff !default;
+$navbar-link-active-background-color: lighten($primary-color, 8%) !default;
+
+//== Panels
+
+$panel-background-color: lighten(#ecf0f1, 4%) !default;
```

Comment:
Color literals like `#ecf0f1` should only be used in variable declarations; they should be referred to via variable everywhere else.